public/docs/ts/latest/tutorial/toh-pt5.jade

```diff
@@ -2,7 +2,7 @@
 
 block includes
   include ../_util-fns
-  - var _appRoutingTsVsAppComp = 'app.routing.ts'
+  - var _appRoutingTsVsAppComp = 'app.module.ts'
```

Comment:
what does this jade syntax do? If its code, we dont use underscore prefixes by convention. If its styling, ignore this :)